frontend/app_flowy/lib/workspace/application/doc/share_bloc.dart

```diff
@@ -33,8 +36,24 @@ class DocShareBloc extends Bloc<DocShareEvent, DocShareState> {
   ExportData _convertDeltaToMarkdown(ExportData value) {
     final result = deltaToMarkdown(value.data);
     value.data = result;
+    writeFile(result);
     return value;
   }
+
+  Future<String> get _localPath async {
+    final dir = await getApplicationDocumentsDirectory();
```

Comment:
Please use appFlowyDocumentDirectory instead. The path is different in different integration modes.

`Future<Directory> appFlowyDocumentDirectory() async {
  Directory documentsDir = await getApplicationDocumentsDirectory();

  switch (integrationEnv()) {
    case IntegrationMode.develop:
      return Directory('${documentsDir.path}/flowy_dev').create();
    case IntegrationMode.release:
      return Directory('${documentsDir.path}/flowy').create();
    case IntegrationMode.test:
      return Directory("${Directory.current.path}/.sandbox");
  }
}`

Reply:
Ok let me get to work on that.

Reply:
Finished applying the changes.